packages/arb-token-bridge-ui/src/components/TransferPanel/AdvancedSettings.tsx

```diff
@@ -252,9 +247,11 @@ export const AdvancedSettings = () => {
                 value={destinationAddress}
                 disabled={inputLocked}
                 spellCheck={false}
-                onChange={e =>
-                  setDestinationAddress(e.target.value?.toLowerCase().trim())
-                }
+                onChange={e => {
+                  const value = e.target.value?.toLowerCase().trim()
```

Comment:
Changed so that `destinationAddress` is `undefined` if empty instead of an empty string.